src/components/App/viewmodel.ts

```diff
@@ -167,15 +165,13 @@ export class AppViewModel {
       || this.video.current!.videoHeight == 0) {
       return;
     }
-    if (this.isMouseEnabled) {
-      if (event.clientY <= window.innerWidth - 110) {
+    if (action == "mouse_move" && !event.buttons) {
+      if (this.isMouseTrackEnabled) {
         document.body.style.cursor = "none";
       } else {
         document.body.style.cursor = "default";
-      }
-    }
-    if (action == "mouse_move" && !this.isMouseTrackEnabled && !event.buttons) {
-      return;
+        return;
+      } 
```

Comment:
If `action == "mouse_move"`, `event.buttons == true`, and `this.isMouseTrackEnabled == false`, then it will send mouse event but will not hide cursor.